packages/testlab/should-as-function.d.ts

```diff
@@ -210,8 +213,8 @@ interface Internal extends ShouldInternal {
   inspect(value: any, obj: any): any;
 }
 
-declare var should: Internal;
-declare var Should: Internal;
+let should: Internal;
+// let Should: Internal;
```

Comment:
I am not able to comprehend the impact of this change where you are removing `declare var`. Is it actually a good idea to lint `.d.ts` files?

Anyhow, if this change works well, then I am ok with it, but please remove the commented-out line.

Reply:
I'll skip `*.d.ts`.